frontend/src/views/Landing.tsx

```diff
@@ -12,9 +42,11 @@ function LandingPage() {
         Join a Game
       </PrimaryButtonLink>
       <br />
-      <TextLink to="/game/create">
+      <TextButton onClick={createNewRoom}>
         Or create a room &#8594;
-      </TextLink>
+      </TextButton>
+      {error ? <ErrorMessage message={error} /> : null}
+      {loading ? <Loading /> : null}
```

Comment:
Quick note on this -- I tested what would happen if I turned the wifi off and tried to **create a room**. It loaded for 45 seconds and then displayed the error message (which is expected). I think this is a good setup, but I wanted to note it down in case you would prefer to do anything else (such as a shorter "end request" time than 45 seconds).

Reply:
Personally I think that should be fine based on testing what LeetCode does in response to no internet (also shows loading for a long time), but I'll add a card to the backlog.